src/query/block/container.go

```diff
@@ -609,3 +609,62 @@ func (it *ucContainerSeriesIter) Next() bool {
 func (it *ucContainerSeriesIter) Current() UnconsolidatedSeries {
 	return it.its[it.idx].Current()
 }
+
+func (b *ucContainerBlock) MultiSeriesIter(
+	concurrency int,
+) ([]UnconsolidatedSeriesIterBatch, error) {
+	if b.err != nil {
+		return nil, b.err
+	}
+
+	if len(b.blocks) == 0 {
+		return nil, nil
+	}
+
+	multiBatches := make([][]UnconsolidatedSeriesIterBatch, 0, len(b.blocks))
+	for _, bl := range b.blocks {
+		batch, err := bl.MultiSeriesIter(concurrency)
+		if err != nil {
+			// NB: do not have to set the iterator error here, since not all
+			// contained blocks necessarily allow mutli series iteration.
```

Comment:
nit: Typo "mutli" -> "multi"

Reply:
Nice catch, may handle that in the cleanup PR if thats ok